app/src/main/kotlin/com/hippo/ehviewer/EhApplication.kt

```diff
@@ -178,6 +179,7 @@ class EhApplication :
                 add(HardwareBitmapInterceptor)
             }
             add(CropBorderInterceptor)
+            add(DetectBorderInterceptor)
```

Comment:
Is it necessary to split crop & detect?
We can remove the `rect` field if we keep them in one interceptor.

Reply:
Yeah, later fastpath on API 29 needs that.